dev/ci/docker/README.md

```diff
@@ -4,10 +4,21 @@ This directory provides Docker images to be used by Coq's CI. The
 images do support Docker autobuild on `hub.docker.com` and Gitlab's
 private registry.
 
-Gitlab CI will build and tag a Docker by default for every job if the
-`SKIP_DOCKER` variable is not set to `false`. In Coq's CI, this
-variable is usually set to `false` indeed to avoid booting a useless
-job.
+Gitlab CI will build and tag a Docker when running a pipeline if either of the
+conditions below is met:
+- If an image with the same name is not available in the
+  [Gitlab container registry](https://gitlab.com/coq/coq/container_registry)
+- If the `SKIP_DOCKER` variable is not set to `false`. In Coq's default CI, this
+  variable is set to `true` precisely to avoid running a lengthy, redundant job.
+
+The first situation can be used to regenerate a fresh Docker image on Gitlab, by
+modifying the `CACHEKEY` value at the first line of the
+[Dockerfile](bionic_coq/Dockerfile) to any unique fresh name, and pushing a
```

Comment:
No, the CACHEKEY in Dockerfile is just a comment, you're supposed to keep it the same as the one in gitlab.yml for human readers but the computers ignore it.